### DISCUSSION TITLE
where can i find the `kube-scheduler.yaml`

### DISCUSSION THREAD

**Discussion by wang-xiaowu** [k3s-io/k3s#8152] [Q&A] [answered]
i am trying to use trimaran instead of the traditional kube-scheduler，
and i wanna backup the `kube-scheduler.yaml` file
where can i find it？

**Comment by brandond** [comment, marked as answer]
There isn't one. The scheduler is run in the main k3s process, the same as all the other control-plane components. It is configured via the --kube-scheduler-arg flag. If you haven't changed that, there is no configuration to back up.
